channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -150,12 +150,8 @@ export class Sender implements IChannelControlsAI {
      */
     private _timeoutHandle: any;
 
-    private _nextPlugin: ITelemetryPlugin;
```

Comment:
These will now be hidden, so any plugin which was relying on these internal fields WILL be broken.
I've added an accessor for the _logger -> log(), but the only *real* usage of the _nextPlugin *should* have been to check and call processTelemetry() which now has a helper processNext()